ece_core/18614.md

```diff
@@ -0,0 +1,19 @@
+# 18-614: Microelectromechanical Systems (MEMS)
+
+## Course Overview
+
+This course covers fabrication and design fundamentals for Microelectromechanical Systems at the micro scale. Expect to learn basic principles from microstructure fabrication, mechanics of silicon and thin-film materials, electrostatic force, capacitive motion detection, fluidic damping, piezoelectricity, piezoresistivity, and thermal micromechanics. 
+
+## What to expect
+
+- **Homework:** Weekly assignments based on the lecture and sometimes examples from the slides provided. Make sure to show all work and reference formulas step by step to maximize your grade.
+- **Exam:** Expect at least one exam as the final, but it has changed year by year and some previous classes had a midterm. Extremely doable, but only if you understand the lectures and make use of the practice exams and homework provided. Also be mindful of time management on the exam as it is a huge factor and at times people may not be able to finish in time.
+- **Final Project:** Usually done in groups 2-4 depending on class size that semester. Usually requires research, applications, and simulation (COMSOL) on MEMS devices discussed in class like resonators, pressure sensors, etc. Requires knowledge of fabrication techniques, mechanical stress equations and other pertinent information from the course. Groups with members from different disciplines helps and is encouraged.
```

Comment:
What is the final project about? Is it just a research paper? I need more detail here.